code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -93,6 +99,38 @@ GLOBAL_LIST_INIT(iron_recipes, list ( \
 	. = ..()
 	recipes = GLOB.sandstone_recipes
 
+// i know there's "sand" in mining module (aka impure silicates aka glass ore), but it doesn't work as a sheet and has some other problems
+// you can remove this if you turn all ores into sheets so they stack, or maybe i'll rework mining module myself
+/obj/item/stack/sheet/mineral/sand
+	name = "sand pile"
+	desc = "A pile of sand that can be smelted into glass or used in sandbags"
+	singular_name = "sand pile"
+	icon = 'icons/obj/mining.dmi'
+	icon_state = "Glass ore"
```

Comment:
Are you sure this is a valid icon_state?

Reply:
I tested it and it works in game. Although the sprite doesn't change depending on the amount of sheets in the stack.